File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Spell.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Spell {

    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("school")
    @Expose
    public Integer school;
    @SerializedName("icon")
    @Expose
    public String icon;
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("rank")
    @Expose
    public String rank;

}

File: app/src/main/java/fr/jbme/raiderioapp/model/RIOCharacter/Spell.kt
package fr.jbme.raiderioapp.model.RIOCharacter

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Spell(
    @SerializedName("id")
    @Expose
    var id: Int? = null,

    @SerializedName("school")
    @Expose
    var school: Int? = null,

    @SerializedName("icon")
    @Expose
    var icon: String? = null,

    @SerializedName("name")
    @Expose
    var name: String? = null,

    @SerializedName("rank")
    @Expose
    var rank: String? = null
)